pkg/inst/tests/test_rdd.R

```diff
@@ -405,6 +405,25 @@ test_that("pipeRDD() on RDDs", {
   expect_equal(actual, expected)
 })
 
+test_that("zipRDD() on RDDs", {
```

Comment:
Yes, I did want to add such test cases. But they are blocked by the bug reported by SPARKR-224, which has been fixed in the sparkr-sql branch but have not yet been ported back to the master branch. I will add such test cases after porting changes from sparkr-sql to master.